documentation/monitoring.md

```diff
@@ -161,8 +161,29 @@ Unfortunately we can't alert which kubernetes service is using aa high number of
 ## Metrics Retention
```

Comment:
I would think this section could move to the Thanos section, or just after, earlier in the document.